.github/guides/VISUALS.md

```diff
@@ -77,10 +77,10 @@ Somewhat like this
 // Lie to the compiler, and pretend we are using a mutable appearance
 // This lets us access vars as expected. Be careful with it tho
 /proc/mutate_icon_state(mutable_appearance/thing) 
-	var/mutable_appearance/temporary_lad = new()
-	temporary_lad.appearance = thing
-	temporary_lad.icon_state += "haha_owned"
-	return temporary_lad.appearance
+ var/mutable_appearance/temporary_lad = new()
```

Comment:
You mention whatever linter we're apparently using doesn't support hard tabs...we need to fix that before we can ever merge a PR that does something like this

Reply:
it's a setting it says under the warning